outbox/README.md

```diff
@@ -8,44 +8,72 @@ Debezium captures the additions to this table and streams the events to consumer
 The receiving service ("shipment-service") receives these events (and would apply some processing based on them),
 excluding any duplicate messages by comparing incoming event ids with already successfully consumed ids.
 
-**Update, March 3, 2019:** Another variant of the receiving service has been added, "shipment-service-quarkus",
-which is functionally the same as the original one, but is implemented using the [Quarkus](https://quarkus.io) stack.
-This allows to build a native binary of that service, resulting in significantly less memory usage and faster start-up than the classic version (based on Thorntail).
+Both services are implemented using the [Quarkus](https://quarkus.io) stack.
+This allows building a native binary of each service, resulting in significantly less memory usage and faster start-up than the JVM-based version.
 
-## Execution
+## Building
 
-Prepare the Java components:
+Prepare the Java components by first performing a Maven build.
 
 ```console
 $ mvn clean install -Pnative -Dnative-image.docker-build=true
 ```
 
+This illustates the usage of Quarkus' `native` profile mode where the quarkus-maven-plugin will generate not only JVM-based artifacts but also native images.
+The `native` profile and `native-image.docker-build` environment variable can be omitted if native image artifacts are not wanted.
+
+NOTE: There is some limitations with the Hibernate ORM Quarkus extension that prohibits running the Debezium Outbox Quarkus extension in native mode, so those arguments should be omitted for now.
```

Comment:
I'd suggest to pull up this note right after line 12. Also is there an issue id we could link to?

Reply:
I followed your advice, adding the two `ReflectiveClassBuildItem` entries we needed in our own extension got past the hurdle, so I've removed this in favor of having native support.